ova/generate_ova.sh

```diff
@@ -102,13 +104,13 @@ build_ova() {
 }
 
 check_version() {
-
-    if [ "${STATUS_PACKAGES}" = "stable" ]; then
-        curl -Isf https://raw.githubusercontent.com/wazuh/wazuh-kibana-app/v${WAZUH_VERSION}-${ELK_VERSION}/README.md > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
-    elif [ "${STATUS_PACKAGES}" = "unstable" ]; then
-        curl -Isf https://packages-dev.wazuh.com/pre-release/app/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
+    if [ "${STATUS}" = "prod" ]; then
+        WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+        curl -Isf https://packages.wazuh.com/${WAZUH_MAJOR}/ui/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
+    elif [ "${STATUS}" = "dev" ]; then
+        curl -Isf https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-1.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
```

Comment:
Hardcoded revision to `1`.

Reply:
Done in 9be1155a0b83ded1a36a106711bad9c9a4191da8